engine/core/IgeEntity.js

```diff
@@ -2021,14 +1940,14 @@ var IgeEntity = IgeObject.extend({
 					this.streamUpdateData([{ effect: type }]);
 				}
 			} else if (ige.isClient) {
-				if (this._pixiContainer && this._pixiContainer._destroyed) {
+				if (!ige.entitiesToRender.trackEntityById[this.id()]) {
```

Comment:
@sloont so is this the proper replacement for `_destroyed` property, checking if entity is "tracked for rendering"?

Reply:
another option is using !this._alive

Reply:
we had discussion previously, but not sure if we had a proper conclusion... I seem to remember that `!_alive` and `_destroyed` don't really represent the same thing

Reply:
yeah @Grenagar and I were looking at this one for a little while. I'm going through the flow step by step right now. Its tough to say. At the very latest, it applies the condition the frame after